example/package.json

```diff
@@ -3,7 +3,7 @@
   "name": "example",
   "version": "0.0.1",
   "dependencies": {
-    "@adobe/gatsby-theme-aio": "*",
+    "@adobe/gatsby-theme-aio": "4.3.0-rc6",
```

Comment:
Can be reverted to`*` ?

Reply:
@dmitrymatio, for some reason, `*` was not working earlier. I kept making code changes and doing another build to see no results until I changed it to match our current version. I'll try again now.

Reply:
Oh, duh...yeah I'll revert. Covid brain...I had to temporarily do that during dev. Doh!